capture/packet.c

```diff
@@ -1070,6 +1102,34 @@ LOCAL MolochPacketRC moloch_packet_ether(MolochPacketBatch_t * batch, MolochPack
 #endif
         return MOLOCH_PACKET_CORRUPT;
     }
+
```

Comment:
definitely should have a config switch that controls this feature and maybe even sets a limit on number of macs to save.

Reply:
# Default Arikime captures the inner mac address, increase this number if also the other mac addresses need to be captured, this increases need for memory (max=7)
#maxMacAddressCount=6

something like this?